git/objects/submodule/base.py

```diff
@@ -1168,11 +1168,11 @@ def iter_items(cls, repo, parent_commit='HEAD'):
 
         for sms in parser.sections():
             n = sm_name(sms)
-            p = parser.get_value(sms, 'path')
-            u = parser.get_value(sms, 'url')
+            p = parser.get(sms, 'path')
+            u = parser.get(sms, 'url')
             b = cls.k_head_default
             if parser.has_option(sms, cls.k_head_option):
-                b = str(parser.get_value(sms, cls.k_head_option))
+                b = str(parser.get(sms, cls.k_head_option))
```

Comment:
could even strip away us of str here... note that `get` is still advisable instead of `get_value` since it could cast+str `1.0` to `1` and then `false` to `False`.